example_sns_init.yaml

```diff
@@ -0,0 +1,117 @@
+# You should make a copy of this file, name it sns_init.yaml, and edit it to
+# your taste.
+#
+# All principal IDs should almost certainly be changed.
+#
+# In this file, 1 year is nominally defined to be 365.25 days.
+#
+# This gets passed to `sns propose`. See propose_sns.sh.
+#
+# This follows the second format developed for the `sns` CLI. The first format
+# will be supported for a time, but this format will eventually become the
+# standard format.
+
+name: Rock Out
+description: >
+    A poem co-written with ChatGPT
+
+    In a realm of code, a beacon forth surges,
+    a wondrous app on the Internet Computer emerges.
+    Born of inspiration divine,
+    A marvel of technology briliantly shines.
+
+    With each line of code, a world takes flight,
+    A symphony of bits, a chorus of bytes.
+    Built on chainkey cryptography, secure and true,
+    An app that transcends, all we once knew...
+
+Principals: [] # TODO: This field will be useful in the future.
```

Comment:
In the (far) future or to be filled out by the user before running `sns propose`?

Reply:
We have a [Jira ticket to implement this](https://dfinity.atlassian.net/jira/software/c/projects/NNS1/issues/NNS1-2293). Not sure when we'll get around to it. If we don't have time before we release 1-proposal, we'll make this field not required, and delete this line.